src/components/CallToContribute.tsx

```diff
@@ -1,82 +1,36 @@
 import React from "react"
-import styled from "@emotion/styled"
 import Link from "./Link"
 import ButtonLink from "./ButtonLink"
-import Icon from "./Icon"
 
 import Translation from "./Translation"
-
-const StyledCard = styled.div`
-  display: flex;
-  background: ${(props) => props.theme.colors.ednBackground};
-  align-items: center;
-  margin-top: 2rem;
-  border: 1px solid ${(props) => props.theme.colors.primary};
-  border-radius: 4px;
-  box-shadow: inset 0 -2px 0 0 ${(props) => props.theme.colors.primary400};
-`
-
-const Column = styled.div`
-  flex: 1 1 50%;
-  padding: 1rem;
-  display: flex;
-  flex-direction: column;
-  color: ${(props) => props.theme.colors.text};
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    text-align: center;
-  }
-`
-
-const ImageColumn = styled(Column)`
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    display: none;
-  }
-`
-
-const GithubButton = styled(ButtonLink)`
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  color: ${(props) => props.theme.colors.background};
-  background-color: ${(props) => props.theme.colors.secondaryButtonBackground};
-  border: 1px solid ${(props) => props.theme.colors.secondaryButtonBorder};
-  &:hover {
-    background-color: ${(props) =>
-      props.theme.colors.secondaryButtonBackgroundHover};
-    color: ${(props) => props.theme.colors.secondaryButtonHoverColor};
-  }
-  &:active {
-    background-color: ${(props) =>
-      props.theme.colors.secondaryButtonBackgroundActive};
-    color: ${(props) => props.theme.colors.secondaryButtonHoverColor};
-  }
-`
-
-const GithubIcon = styled(Icon)`
-  fill: ${(props) => props.theme.colors.background};
-  margin-right: 0.5rem;
-`
-
-const Description = styled.p`
-  line-height: 140%;
-  color: ${(props) => props.theme.colors.text};
-  font-family: ${(props) => props.theme.fonts.monospace};
-`
-
-const Title = styled.h2`
-  font-family: ${(props) => props.theme.fonts.monospace};
-  text-transform: uppercase;
-  background: ${(props) => props.theme.colors.border};
-  padding: 0.25rem;
-`
+import { Flex, Heading, Icon, Text } from "@chakra-ui/react"
+import { FaGithub } from "react-icons/fa"
 
 export interface IProps {
   editPath: string
 }
 
 const CallToContribute: React.FC<IProps> = ({ editPath }) => (
-  <StyledCard>
-    <ImageColumn>
+  <Flex
+    bg="ednBackground"
+    alignItems="center"
+    mt={8}
+    borderWidth="1px"
+    borderStyle="solid"
```

Comment:
This is the same as these two lines
```suggestion
    border="1px"
```